vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -282,17 +289,98 @@ def parse_container_definition(container_def, pod_id)
 
     def parse_container(container, pod_id)
       {
-        :type          => 'ContainerKubernetes',
-        :ems_ref       => "#{pod_id}_#{container.name}_#{container.image}",
-        :name          => container.name,
-        :image         => container.image,
-        :restart_count => container.restartCount,
-        :backing_ref   => container.containerID,
-        :image_ref     => container.imageID
+        :type            => 'ContainerKubernetes',
+        :ems_ref         => "#{pod_id}_#{container.name}_#{container.image}",
+        :name            => container.name,
+        :image           => container.image,
+        :restart_count   => container.restartCount,
+        :backing_ref     => container.containerID,
+        :image_ref       => container.imageID,
+        :container_image => parse_container_image(container.image, container.imageID)
```

Comment:
@zeari from what I see here you always create new `container_image` entities?
You should instead identify the image and store it only once (one-to-many, not one-to-one with repeated images).